geomesa-quickstart-python/pyJavaClasses/SimpleFeatureType.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+#coding:utf-8
+"""
+SimpleFeatureType.py
+
+Description:
+   This function uses jnius to implement various features from the SimpleFeatureType(s)
+   java classes.
+
+Created by: Jordan Muss
+
+Creation Date: 4-4-2017
+Version:       1.0
+
+Dependencies: 
+         Public:     jnius
+         Private:   setupJnius
+
+Interfaces:
+        createSimpleFeatureType              Class to create a 'bare' simpleFeatureType from a list of attributes,
+                                                              and to perform appropriate tasks, e.g. set feature indices
+
+Updates:
+
+To Do:
+"""
+class simpleFeatureType():
```

Comment:
PEP-8 says [CapWords for class names](https://www.python.org/dev/peps/pep-0008/#class-names)